doc/api/test.md

```diff
@@ -1579,9 +1579,10 @@ added:
 Enables timer mocking for the specified timers.
 
 * `timers` {Array} An optional array containing the timers to mock.
-  The currently supported timer values are `'setInterval'` and `'setTimeout'`.
-  If no array is provided, all timers (`'setInterval'`, `'clearInterval'`, `'setTimeout'`,
-  and `'clearTimeout'`) will be mocked by default.
+  The currently supported timer values are `'setInterval'`, `'setTimeout'`
+  and `'setImmediate'`.  If no array is provided, all timers (`'setInterval'`,
+  `'clearInterval'`, `'setTimeout'`, `'clearTimeout'`, `'setImmediate'`,
+  and `'clearImmediate'`) will be mocked by default.
```

Comment:
I'm not sure if `clearInteval`, `clearTimeout` and `clearImmediate` should be mentioned here as they technically are not timers.

And might confuse the person reading the docs as "why can I provide A,B,C" but if I provide nothing then "A,B,C,D,E,F" get mocked?

Reply:
I always associated these as utility functions and not actual timers per-se. Maybe I have a wronged understanding of the concept of "timer" at play here.

Reply:
They are all timers because they deal with time and return a timer ID, and have a set/clear API.

Reply:
Gotcha, well, in that definition of timer that does make sense. But for the average audience that _can_ be confusing. It would probably make sense to somewhere in the docs mention that these are considered timers due to, for example, what you mentioned 👀

Reply:
wdyt @ljharb

Reply:
I would argue that the average audience is familiar with the many-decades old concept of "timers" from the web, and that assuming anything *else* would be confusing.

Reply:
Fair enough. Thanks for your time, @ljharb :)

Reply:
(to be clear, it's always great to provide a note for any points of confusion; i just think the default here should be that "timers" includes all 6 of these functions)

Reply:
btw, would you guys review and check if it's ready to go? @ljharb @ovflowd thanks!